### DISCUSSION TITLE
Ion button with dropdown - loading state

### DISCUSSION THREAD

**Discussion by allan-chagas-brisa** [iurynogueira/ion#1163] [Ideas]
Discussion created so we can establish the ion button's loading behavior with a dropdown and its variations.

**Comment by allan-chagas-brisa** [comment]
known variations: 
https://main--62eab350a45bdb0a5818520e.chromatic.com/?path=/story/ion-navigation-button--with-dropdown
https://main--62eab350a45bdb0a5818520e.chromatic.com/?path=/story/ion-navigation-button--with-dropdown-multiple-choices

**Comment by Jeter-Megaron** [comment]
In the case of the button load with a select, I support replacing the icon while it is loading. Regarding the variant of the select button with a dropdown and a counter, I believe it is not a relevant element for the Design System. However, a use case would be to replace the 'down' icon with the counter after selecting the first item. 
![unnamed](https://github.com/user-attachments/assets/edb400ef-a08a-4635-a404-8facaef07dd4)
